pkg/api/v1/types.go

```diff
@@ -1168,6 +1168,12 @@ type ContainerStatus struct {
 	// Container's ID in the format 'docker://<container_id>'.
 	// More info: http://releases.k8s.io/HEAD/docs/user-guide/container-environment.md#container-information
 	ContainerID string `json:"containerID,omitempty"`
+	// The image creation time from docker image inspection
+	ImageCreationTime unversioned.Time `json:"imageCreationTime,omitempty"`
+	// The image size time from docker image inspection
```

Comment:
This can't be defined in terms of what docker inspect returns.

What's the difference between size and virtual size?